comp_loinc/build.py

```diff
@@ -1,83 +1,141 @@
-import typer
-from typing import List
-from ingest.part_ingest import PartOntology
-from ingest.code_ingest import CodeIngest
+"""CLI
+
+Resources
+- CLI type system docs (typer is a higher leve wrapper around click): https://click.palletsprojects.com/en/8.1.x/api/
```

Comment:
@putmantime I found out that `typer` uses the popular CLI library `click` behind-the-scenes. From what I've seen so far, if we want to tweak CLI functionality, the `click` docs will often be the better source of information.